supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -88,15 +88,22 @@ case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, en
 
       def networkFromLabels(vpcConfig: VirtualPrivateCloudConfiguration, projectLabels: ProjectLabels): Network = {
         val networkLabelOption = projectLabels.labels.find(l => l._1.equals(vpcConfig.name))
+        val subnetworkLabelOption = vpcConfig.subnetwork.flatMap(s => projectLabels.labels.find(l => l._1.equals(s)))
 
-        networkLabelOption match {
-          case Some(networkLabel) =>
+        (networkLabelOption, subnetworkLabelOption) match {
+          case (Some(networkLabel), Some(subnetworkLabel)) =>
             new Network()
               .setUsePrivateAddress(createPipelineParameters.runtimeAttributes.noAddress)
               .setName(VirtualPrivateCloudNetworkPath.format(createPipelineParameters.projectId, networkLabel._2))
-          case None =>
-            jobLogger.debug("Project `{}` does not have high security network configured. " +
-              "Project metadata does not have network label key `{}`. Falling back to running the job on default network.", createPipelineParameters.projectId: Any, vpcConfig.name: Any)
+              .setSubnetwork(subnetworkLabel._2)
+
+          case (Some(networkLabel), None) =>
```

Comment:
There's a fair amount of repetition between these cases. I think you could simplify with something like:
```scala
          case (Some(networkLabel), subnetworkLabel) =>
            val network = new Network()
              .setUsePrivateAddress(createPipelineParameters.runtimeAttributes.noAddress)
              .setName(VirtualPrivateCloudNetworkPath.format(createPipelineParameters.projectId, networkLabel._2))
            
            subnetworkLabel foreach { case (_, subnet) => network.setSubnetwork(subnet) }
            network
```

Reply:
Is it possible to update `val network` object's property once created?

Reply:
Clicking through to the implementation of the function, it looks like it's just a normal mutable java class behind the scenes

Reply:
Ahh I see. That's right. I have removed the repetition. Thank you.